app/Traits/GetInvoices.php

```diff
@@ -30,15 +30,7 @@ public function getTodaysInvoices(Admin $admin) {
 	public function getWeeklyInvoices(Admin $admin) {
 
 		// 0 is Monday, 1 is Tuesday, etc.
-		$invoices_array = [
-			1 => 0,
-			2 => 0,
-			3 => 0,
-			4 => 0,
-			5 => 0,
-			6 => 0,
-			7 => 0,
-		];
+		$invoices_array = array_fill(0, 7, 0);
```

Comment:
The comment said it went 0 --> 6 but then the array was defined 1 --> 7.  This was causing an error when running, so I made it go 0 --> 6, and threw it into a one line function instead to generate the array for us.